content/guides/anomalies.md

```diff
@@ -73,6 +73,22 @@ We currently offer four different anomaly detection algorithms.
 
 All of the seasonal algorithms may use up to a couple of months of historical data when calculating a metric's expected normal range of behavior. By using a significant amount of past data, the algorithms are able to avoid giving too much weight to abnormal behavior that might have occurred in the recent past.
 
+The figures below illustrate how and when these four algorithms behave differently from one another. In the first figure, _basic_ will successfully identify anomalies that spike out of the normal range of values, but it does not incorporate the repeating, seasonal pattern into its predicted range of values. By contrast, _robust_, _agile_, and _adaptice_ all recognize the seasonal pattern and can detect more nuanced anomalies (e.g., if the metric was to flatline near its minimum value).
+
+<img src="/static/images/anomalies/alg_comparison_1.png" style="width:100%; border:1px solid #777777"/>
+
+In the next figure, the metric exhibits a sudden level shift. _Agile_ and _adaptive_ adjust more quickly to the level shift than does _robust_. Also, the width of _robust_'s bounds increases to reflect greater uncertaintly after the level shift; the width of _agile_ and _adaptive_ bounds remains unchanged. _Basic_ is clearly a poor fit for this scenario, where the metric exhibits a strong weekly seasonal pattern.
+
+<img src="/static/images/anomalies/alg_comparison_2.png" style="width:100%; border:1px solid #777777"/>
+
+The next figure shows how the algorithms react to an hour-long anomaly. _Robust_ completely ignores this anomaly. All the other algorithms start to behave as if the anomaly is the new normal. _Agile_ and _adaptive_ even identify the return to the metric to its original level as an anomaly.
```

Comment:
Totally minor edit, but "the return to the metric to its original level" reads a bit awkwardly (I had to read it twice to grok it). Maybe it could be simplified to "the metric's return to its original level".

Otherwise, this is a fantastic addition and illustrating the difference is extremely helpful!